test/Libraries/TestServices/TestSessionConfiguration.cs

```diff
@@ -48,7 +48,7 @@ public TestSessionConfiguration(string dynamoCoreDirectory, string configFileDir
             if (!File.Exists(configPath))
             {
                 DynamoCorePath = dynamoCoreDirectory;
-                RequestedLibraryVersion = LibraryVersion.Version219;
+                RequestedLibraryVersion = LibraryVersion.Version220;
```

Comment:
does this mean all of our Dynamo geometry tests have been running on ASM 219 for all versions? 

does the CI have asm 219?  - I guess we should just be looking out for failing tests.